tests/test_create_climo_files.py

```diff
@@ -158,22 +162,46 @@ def test_climo_metadata(outdir, tiny_dataset, operation, t_start, t_end, split_v
 
     suffix = {
         'std': 'SD',
-        'mean': 'Mean'
+        'mean': 'Mean',
+        'sum': 'Mean'
     }[operation]
 
     if split_intervals:
         assert frequencies == {
            'daily': {'mClim' + suffix, 'sClim' + suffix, 'aClim' + suffix},
            'monthly': {'sClim' + suffix, 'aClim' + suffix},
+           'seasonal': {'sClim' + suffix, 'aClim' + suffix},
            'yearly': {'aClim' + suffix},
         }[tiny_dataset.time_resolution]
     else:
         assert frequencies == {
             'daily': {'msaClim' + suffix},
             'monthly': {'saClim' + suffix},
```

Comment:
Same as above, should it be `'monthly': {'mClim' + suffix}`?

Reply:
Same as above: Nope, it should be 'monthly': {'msaClim' + suffix}, for the same regions. Same question about testing.